target/linux/ramips/mt7621/base-files/etc/uci-defaults/99_fwenv-store-ethaddr.sh

```diff
@@ -0,0 +1,15 @@
+[ ! -e /etc/fw_env.config ] && exit 0
+
+. /lib/functions/system.sh
+
+case "$(board_name)" in
+dna,valokuitu-plus-ex400)
+	macaddr=$(/usr/sbin/fw_printenv -n ethaddr 2>/dev/null)
+	mac2addr=$(/usr/sbin/fw_printenv -n eth1addr 2>/dev/null)
```

Comment:
nitpick: naming with **eth1**addr vs **mac2**addr is a bit inconsistent confusing.

Reply:
yes, I'll change it.